src/strategies/erc20-votes-with-override/index.ts

```diff
@@ -5,12 +5,18 @@ import { multicall } from '../../utils';
 export const author = 'serenae-fansubs';
 export const version = '0.1.0';
 
-const getVotesName = "getVotes";
-const getVotesABI = ["function getVotes(address account) view returns (uint256)"];
-const balanceOfName = "balanceOf";
-const balanceOfABI = ["function balanceOf(address account) view returns (uint256)"];
-const delegatesName = "delegates";
-const delegatesABI = ["function delegates(address account) view returns (address)"];
+const getVotesName = 'getVotes';
```

Comment:
Hi @nginnever Please revert changes from other strategies other than new strategy :) and please update your branch with latest changes from master, thanks

Reply:
I think this just come from the lint